dartsim/src/SDFFeatures.cc

```diff
@@ -829,7 +829,21 @@ Identity SDFFeatures::ConstructSdfCollision(
       math::Vector3d fdir1 = frictionDirectionElem->Get<math::Vector3d>();
       aspect->setFirstFrictionDirection(math::eigen3::convert(fdir1));
 
-      const std::string kExpressedIn = "ignition:expressed_in";
+      std::string kExpressedIn = "gz:expressed_in";
```

Comment:
nit: this is no longer `const` so I would rename `kExpressedIn` to `expressedIn`

Reply:
Fixed! https://github.com/gazebosim/gz-physics/pull/356/commits/3af25face1afdf8aff13a21fc228d6062c4c2e40